code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -208,6 +208,16 @@
 	required_reagents = list(/datum/reagent/medicine/omnizine/protozine = 1, /datum/reagent/water/holywater = 1, /datum/reagent/toxin/mutagen = 1)
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_HEALING | REACTION_TAG_PLANT | REACTION_TAG_OTHER
 
+/datum/chemical_reaction/medicine/fishy_reagent
+	results = list(/datum/reagent/medicine/strange_reagent/fishy_reagent = 3)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/water/salt = 1, /datum/reagent/toxin/carpotoxin = 1)
+	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_HEALING | REACTION_TAG_OTHER
+
+/datum/chemical_reaction/medicine/fishy_reagent/alt
+	results = list(/datum/reagent/medicine/strange_reagent/fishy_reagent = 3)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/water/salt = 1, /datum/reagent/toxin/tetrodotoxin = 1)
```

Comment:
I suppose the required TTX could be half as much given it's a bit rarer than carpotoxin.

Reply:
made it fgive more